assessment_store/tasks/db_tasks.py

```diff
@@ -149,6 +148,12 @@ def seed_local_assessment_store_db(c):
                     "minimum_score": 0,
                     "maximum_score": 3,
                 },
+                {
```

Comment:
I'm not sure how this works, but it looks like the scoring systems are only added if they don't exist first? And at the moment it only checks for OneToFive and ZeroToThree ... so maybe there needs to be a check for ZeroToOne?

Reply:
Good spot 👍, will add a check for ZeroToOne as well.